src/app/auth/init/establishState.ts

```diff
@@ -10,7 +11,7 @@ const initializer: Initializer = async({dispatch, userLoader}) => {
   const user = await userLoader.getCurrentUser();
 
   if (user) {
-    dispatch(receiveUser(formatUser(user)));
+    dispatch(receiveUser(formatUser(user as AccountsUser)));
```

Comment:
This is might be a @TomWoodward question, but if we need to narrow this, it might make sense to replace REX's `AccountsUser` with `ApiUser` from `ts-utils`